.travis.yml

```diff
@@ -8,4 +8,4 @@ before_install:
   - pip install cython
 script:
   - cd $TRAVIS_BUILD_DIR
-  - python setup.py test
+  - python setup.py test --cython
```

Comment:
This doesn't change what was done previously, as `python setup.py [cmd]` will now try to use an existing `pybloomfilter.c`